internal/controller/driver_controller.go

```diff
@@ -613,9 +621,9 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 								Image:           r.images["addons"],
 								ImagePullPolicy: imagePullPolicy,
 								Args: append(
-									slices.Clone(leaderElectionArgs),
+									slices.Clone(csiAddonsLeaderElection),
```

Comment:
if these args are used only once, here, there is no need to define it on top and clone it here. Just have it directly here in the first place